kornia/geometry/vector.py

```diff
@@ -19,6 +19,12 @@ def __init__(self, vector: Tensor) -> None:
         super().__init__(vector)
         KORNIA_CHECK(vector.shape[-1] == 3)
 
+    def __repr__(self) -> str:
+        return f"x: {self.x}\ny: {self.y}\nz: {self.z}"
+
+    def __getitem__(self, idx) -> "Vector3":
+        return Vector3(stack((self.data[idx, 0], self.data[idx, 1], self.data[idx, 2])))
```

Comment:
needs stack here? Can't be just a slice of data?
```suggestion
        return Vector3(self.data[idx, ...])
```

Reply:
oh right! Silly mistake  :man_facepalming: